best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
```

Comment:
Brad was IINM quite opposed to this bit of received wisdom for performance reasons.

I go back and forth but often, as long at your expects are in a logical order, it seems like it shouldn't matter.